client/.eslintcache

```diff
@@ -1 +1 @@
-[{"/Users/anthonystematz-breitling/Desktop/Code/Cerbanimo/cerbanimo/client/src/index.js":"1","/Users/anthonystematz-breitling/Desktop/Code/Cerbanimo/cerbanimo/client/src/App.js":"2","/Users/anthonystematz-breitling/Desktop/Code/Cerbanimo/cerbanimo/client/src/reportWebVitals.js":"3","/Users/anthonystematz-breitling/Desktop/cerbanimo/client/src/index.js":"4","/Users/anthonystematz-breitling/Desktop/cerbanimo/client/src/App.js":"5","/Users/anthonystematz-breitling/Desktop/cerbanimo/client/src/reportWebVitals.js":"6"},{"size":500,"mtime":1609777455841,"results":"7","hashOfConfig":"8"},{"size":528,"mtime":1609777455838,"results":"9","hashOfConfig":"8"},{"size":362,"mtime":1609777455842,"results":"10","hashOfConfig":"8"},{"size":500,"mtime":1609777455841,"results":"11","hashOfConfig":"12"},{"size":674,"mtime":1609791371866,"results":"13","hashOfConfig":"12"},{"size":362,"mtime":1609777455842,"results":"14","hashOfConfig":"12"},{"filePath":"15","messages":"16","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"17"},"1s3o44e",{"filePath":"18","messages":"19","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"20"},{"filePath":"21","messages":"22","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"17"},{"filePath":"23","messages":"24","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"1qjmmgf",{"filePath":"25","messages":"26","errorCount":0,"warningCount":1,"fixableErrorCount":0,"fixableWarningCount":0,"source":null},{"filePath":"27","messages":"28","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"/Users/anthonystematz-breitling/Desktop/Code/Cerbanimo/cerbanimo/client/src/index.js",[],["29","30"],"/Users/anthonystematz-breitling/Desktop/Code/Cerbanimo/cerbanimo/client/src/App.js",[],["31","32"],"/Users/anthonystematz-breitling/Desktop/Code/Cerbanimo/cerbanimo/client/src/reportWebVitals.js",[],"/Users/anthonystematz-breitling/Desktop/cerbanimo/client/src/index.js",[],"/Users/anthonystematz-breitling/Desktop/cerbanimo/client/src/App.js",["33"],"/Users/anthonystematz-breitling/Desktop/cerbanimo/client/src/reportWebVitals.js",[],{"ruleId":"34","replacedBy":"35"},{"ruleId":"36","replacedBy":"37"},{"ruleId":"34","replacedBy":"38"},{"ruleId":"36","replacedBy":"39"},{"ruleId":"40","severity":1,"message":"41","line":5,"column":5,"nodeType":"42","messageId":"43","endLine":5,"endColumn":20},"no-native-reassign",["44"],"no-negated-in-lhs",["45"],["44"],["45"],"no-unused-vars","'isAuthenticated' is assigned a value but never used.","Identifier","unusedVar","no-global-assign","no-unsafe-negation"]
```

Comment:
Do we need this to make its way onto GitHub, or should we add to .gitignore?

Reply:
I do not even know where that file came from, I will look into it -- it will either be deleted or .gitignore

Reply:
cache file has been deleted, occurred due to moving the eslint from one file to another